src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -83,19 +113,15 @@ const getIssuer = (token: string, logger: Logger): string | undefined => {
 	return unverifiedClaims.iss;
 };
 
-const verifySymmetricJwt = (token: string, secret: string, req: Request, tokenType: TokenType, logger: Logger): boolean => {
+const verifySymmetricJwt = (token: string, secret: string, logger: Logger) => {
```

Comment:
Maybe the method name should be getDecodedClaimes , decodeCliams, etc, instead of verifyXXX as it doesn't verify?